gnirehtet.cmd

```diff
@@ -0,0 +1,150 @@
+@ECHO OFF
+CALL :locate_file ADB %~dp0adb.exe "%LOCALAPPDATA%\Android\sdk\platform-tools\adb.exe"
```

Comment:
BTW, you can add any path you want to the same line (even %ADB% for system variable) and it'll look up for file there.